### ISSUE TITLE
Forge Connection Closed: mismatched mod channel list?

### ISSUE THREAD

**Issue by tianxiu2b2t** [LuckPerms/LuckPerms#3954]
### Description

When I connected Forge 1.21 server with luckperms(latest). And then disconnect cause by: mismatched mod channel list.
I found any error messages, I thought it's luckperms error

### Reproduction Steps

Luckperms Forge 5.4.137

### Expected Behaviour

[098月2024 11:58:45.334] [Netty Server IO #3/ERROR] [net.minecraftforge.network.NetworkRegistry/NETREGISTRY]: Channels [luckperms:update] rejected their client side version number
[098月2024 11:58:45.336] [Netty Server IO #3/ERROR] [net.minecraftforge.network.ForgePacketHandler/FORGE_PACKET_HANDLER]: Terminating connection with client, mismatched channel list
[098月2024 11:58:45.349] [Server thread/INFO] [net.minecraft.server.network.ServerConfigurationPacketListenerImpl/]: com.mojang.authlib.GameProfile@41eff0ab[id=1e5ceee8-1d03-34c9-ae21-a8ec35e5e92a,name=egg,properties={}] lost connection: Connection closed - mismatched mod channel list

### Server Details

Forge mod loading, version 51.0.30, for MC 1.21 with MCP 20240613.102039

### LuckPerms Version

v5.4.137

**Comment by BNTFryingPan** [comment]
do you have LP installed on the client? if so, remove it

**Comment by Crosus97** [comment]
you fix this problem? @xProxyRed

**Comment by xProxyRed** [comment]
> you fix this problem? @xProxyRed

no, I simply switched from Forge to Fabric. everything runs smoothly there.
